### PR TITLE
[IMP] website: redirects

### PR DESCRIPTION
**Pull request by Donapi**
[taskid-3667291](https://www.odoo.com/web#id=3667291&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/websites/website/pages.rst [hunk 1]
```diff
@@ -7,23 +7,22 @@ Pages
 Odoo allows you to create pages for your website and customize their content and appearance to your
 needs.
 
-.. note::
-   *Static* pages are pages that have stable content, such as the homepage. You can manually create
-   new ones, define their URL, adapt their :ref:`properties <website/page_properties>`, etc.
-   *Dynamic* pages, on the other hand, are generated dynamically. All pages generated automatically
-   by Odoo, for example when you install an app or module (e.g., `/shop` or `/blog`) or publish a
-   new product or blog post, are dynamic pages and are therefore managed differently.
+**Static** pages have stable content, such as the homepage. You can manually create new ones, define
```
Comment (auva-odoo) [inline content/applications/websites/website/pages.rst:10]: Didn't you want to add an anchor here so you could reference this paragraph elsewhere in tsi doc where we talk about static and dynamic pages?

**Review by auva-odoo — CHANGES_REQUESTED** [review]
There you go @Donapi :slightly_smiling_face: We can talk about it tomorrow if you'd like.
One thing though, I think that in this case, the changes should be applied from 16.0 instead of 17.0 since nothing has changed functionnally speaking between these versions (which also means it would be pretty easy to do, with very little additionaly effort).

File: content/applications/websites/website/pages.rst [hunk 2]
```diff
@@ -152,36 +151,38 @@ Redirects`.
...
 
 Toggle the :guilabel:`Activate` switch to deactivate the redirection.
 
-.. note::
-   404, 301 and 302 redirections only work if the original page has been :ref:`unpublished
-   <website/un-publish-page>` or :ref:`deleted <website/delete-page>`.
+.. important::
+   404, 301, and 302 redirections are meant to migrate traffic from
+   :ref:`unpublished <website/un-publish-page>` or :ref:`deleted <website/delete-page>` pages
+   to *new* pages, while the 308 redirect is used for *permanent* redirections of *existing* pages.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages.rst:185]: The aim of having the information in an important note is to clearly distinguish the types of redirections 404, 301, and 302 vs. 308, so I think it makes sense to keep it in this note. I can also include it for each type of redirection, but then the information would be duplicated.

File: content/applications/websites/website/pages.rst (hunk 1 above)
Comment (Donapi) [reply content/applications/websites/website/pages.rst:10]: At first I thought it was clear enough as clarified in the intro but then I added the anchors

File: content/applications/websites/website/pages.rst [hunk 3]
```diff
@@ -152,36 +151,38 @@ Redirects`.
...
-     search engine results.
-   - :guilabel:`308 Redirect / Rewrite`: for permanent redirections of dynamic pages; the new URL is
-     shown in search engine results and the redirect is cached by browsers. Use this redirect type
-     to rename a dynamic page, for example, if you wish to rename `/shop` into `/market`.
+     redesigning or updating the page. The new URL is neither cached by browsers nor shown in search
+     engine results.
+   - :guilabel:`308 Redirect/Rewrite`: for permanent redirections of dynamic pages; the URL is
+     renamed and shown in search engine results and is cached by browsers. Use this redirect type
+     to rename a dynamic page. For example, if you wish to rename /shop into /market. The 308
+     redirect rewrites the URL link but do not show any new view/resource.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/pages.rst]: I removed this last sentence that is indeed not clear...

**Review by auva-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**